tools/oeedger8r/src/Headers.ml

```diff
@@ -173,9 +173,6 @@ let generate_args (ec : enclave_content) =
     "#ifndef " ^ guard_macro;
     "#define " ^ guard_macro;
     "";
-    "#include <stdint.h>";
-    "#include <stdlib.h> /* for wchar_t */";
-    "";
     include_errno;
```

Comment:
This seems like a breaking change.
Do user written edl files that use wchart_t now have to explicitly do this include?

Reply:
see above. I'll split it out.